InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -127,4 +132,23 @@ public void rotateToDeg(double targetPoint) {
         targetChange = 0;

```

Comment:
it seems like targetChange is only being used by the rotate(spinRate) and periodic() methods. So as of right now, rotateToDeg() will just hold the position (periodic changes the position by a multiple of targetChange). It seems like we'll need another state machine for auto aligning, manual aligning, or holding still, and periodic() would choose what to do given the state (aka if auto aligning, call rotateTurretGivenLLOffset, if manual call rotate(spinRate), if holding then do nothing)

Reply:
we can discuss this implementation at the meeting today

Reply:
also don't forget to add the gear ratio on line 130!